docs/modules/ROOT/pages/interfaces.adoc

```diff
@@ -0,0 +1,177 @@
+= Interfaces and Dispatchers
+
+This section describes the interfaces OpenZeppelin Contracts for Cairo offer, and explains the design choices behind them.
+
+Interfaces can be found in the module tree under the `interface` submodule, such as `token::erc20::interface`. For example:
+
+```javascript
+use openzeppelin::token::erc20::interface::IERC20Dispatcher;
+```
+
+or
+
+```javascript
+use openzeppelin::token::erc20::dual20::DualERC20Trait;
+```
+
+NOTE: For simplicity, we'll use ERC20 as example but the same concepts apply to other modules.
+
+== Interface types
+The library offers three types of traits:
+
+=== Interface traits (`IERC20`)
```

Comment:
```suggestion
=== Interface traits
```

I think it looks cleaner without it, and the trait name is already present in the example.